css-backgrounds-4/Overview.bs

```diff
@@ -223,7 +229,10 @@ Painting Area: the 'background-clip' property</h3>
 		<pre class=lang-css>
 		.foo {
 			border: 30px solid;
-			border-color: skyblue orange yellowgreen indianred, black yellow;
+			border-top-color: stripes(skyblue, black);
+			border-right-color: stripes(orange, yellow);
+			border-bottom-color: stripes(yellowgreen, black);
+			border-left-color: stripes(indianred, yellow);
```

Comment:
I believe this example is trying to show off the use of the 'border-color' shorthand, this rewrite into longhands isn't appropriate.

Reply:
Ok, I'll change it back to `border-color`.

Sebastian

Reply:
Done.